README.md

```diff
@@ -571,6 +580,10 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Superb AI](https://superb-ai.com) - ML DataOps platform providing various tools to build, label, manage and iterate on training data.
 * [Syndicai](https://syndicai.co) - Easy-to-use cloud agnostic platform that deploys, manages, and scales any trained AI model in minutes with no configuration & infrastructure setup.
 * [Talend Studio](https://www.talend.com/) - Data integration platform that provides various software and services for data integration, data management, enterprise application integration, data quality, cloud storage and Big Data.
+* [Tecton](https://www.tecton.ai/) - Tecton is an all-in-one system to build, automate, and centralize feature workflows for production ML.
 * [Valohai](https://valohai.com/) - Machine orchestration, version control and pipeline management for deep learning.
 * [Vertex AI](https://cloud.google.com/vertex-ai) - Vertex AI Workbench is the single environment for data scientists to complete all of their ML work, from experimentation, to deployment, to managing and monitoring models. It is a Jupyter-based fully managed, scalable, enterprise-ready compute infrastructure with security controls and user management capabilities.
+* [Ultralytics](https://ultralytics.com/) Ultralytics simplifies data sourcing, labeling, model training, and deployment services that offer artificial intelligence.
```

Comment:
Seems only has 9 employees on linkedin, great to support startups by adding to the list but keen ot keep an eye on these as we've had cases in the past where these disappear or change their domain very drastically

Reply:
Sure, but this is a popular startup recently along with generative AI.

Reply:
There's going to be a lot of popular GenAI startups in the next couple of months, this goes back to the https://github.com/EthicalML/awesome-production-machine-learning/blob/master/CONTRIBUTING.md guidelines that we're not trying to make this an `This is not an exhaustive listing` but a curated listing. I think adding to the CONTRIBUTING.md that startups should have some thresholds similar to OSS projects (ie min 15 employees, and 1 year going)

Reply:
Ok ok, but if you take a look at their OSS project profile: their YOLOv[4-8] has a very high impact in the CV domain and the contributors are a lot (~50), the star number seems good too (~4k). @axsaucedo I think Ultralytics is indeed a company to be written a bit here.
![image](https://user-images.githubusercontent.com/8592144/219962286-ba4b41cd-e961-43ed-9a8a-fecb050f33e1.png)

Reply:
Ok that's actually great to hear - similar to the contributing.md there will certainly be exceptions, and this seems like one of them, appreciate providing these overviews. LGTM